sass/style.scss

```diff
@@ -181,14 +181,22 @@ body > main {
 }
 
 /* Menu item styling */
```

Comment:
we can use scss to wrap all the #site-navigation descendent in {} and avoid repeating selectors

Reply:
great point, thank you. I was still in vanilla css mode :/